.travis.yml

```diff
@@ -23,6 +17,8 @@ _android_job_template: &android_job_template
   env:
     global:
       - GRADLE_OPTS="-Xms128m"
+      - ABI="default;armeabi-v7a"
+      - ADB_INSTALL_TIMEOUT=10 # minutes
```

Comment:
Don't we want do decrease this timeout threshold a bit?

Reply:
This line can be completely removed, we're not launching an emulator - my bad.